packages/react-native-payments/lib/js/NativePayments.js

```diff
@@ -16,6 +16,7 @@ const NativePayments: {
   show: () => Promise<any>,
   abort: () => Promise<any>,
   complete: PaymentComplete => Promise<any>,
+  availableNetworks: () => Promise<any>,
```

Comment:
@chukohsin 

This could probably be typed as an array of strings, or more specifically:

```
availableNetworks: () => Promise<PKPaymentNetwork[]>
```

Where PKPaymentNetwork = 
```
types PKPaymentNetwork = 'amex' | 'cartesBancaires' | 'masterCard' | ....
```

Reply:
@CuriousMark Agree on `string[]` and will update it. Didn't want to use specific type because it depends on what we get back from `NativePayments.availableNetworks`. It seems to change with version.